homework/week2/2025320048-yujuyeon/1918번.c

```diff
@@ -0,0 +1,147 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+typedef struct {
+    int* data;
+    int n;
+    int top;
+} Stack;
+void init(Stack* s) {
+    s->data = (int*)malloc(sizeof(int) * s->n);
```

Comment:
함수명이 init인데, 이 함수 호출전에 n이라는 변수가 설정되어있어야하는 점이 이상합니다.
아래와 같은 스타일은 어떠신가요?
```suggestion
void init(Stack* s, int stack_size) {
    s -> n = stack_size;
    s->data = (int*)malloc(sizeof(int) * stack_size);
```